src/main/java/hannah/task/Event.java

```diff
@@ -0,0 +1,37 @@
+package hannah.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+    String name;
+    LocalDate startDuration;
+    LocalDate endDuration;
+    public Event(String name){
+        super(name);
+
+    }
+    @Override
+    public String toString(){
+        return("[E]" + super.toString() + " from " +  this.startDuration + " to " + this.endDuration);
+    }
```

Comment:
Not a coding standard issue, perhaps you will want to separate line 10 to 11 and line 14 to 15.